change_rf_for.py

```diff
@@ -22,11 +22,22 @@ def next_lines_in_block(lst):
 
     with open(name, mode='r', newline='', encoding='utf8') as fp:
         content = fp.readlines()
+
+        # skip empty files
+        if not content:
+            return False
+
         line_ending = '\r\n' if '\r\n' in content[0] else '\n'    # no special treatment for Mac ;)
-        in_block = False
 
+        in_block = False
```

Comment:
Does this come from the linter? Cause `in_block` is set a few lines above, just before starting the context?

Reply:
it is actually the for_position which wasn't set..